contracts/BitmaskRBAC.sol

```diff
@@ -1,27 +1,32 @@
 pragma solidity ^0.4.22;
 
-import "openzeppelin-solidity/contracts/ownership/rbac/RBAC.sol";
+import "./Bitmask.sol";
 
 /**
 * @title Role-Based Access Control (RBAC) supporting bitmasks, for getting/setting multiple roles atomically
 * @author TruSet
-* @dev Enhance openzeppelin's RBAC to allow getting and setting of multiple roles in a single
+* @dev Use a bitmask to allow getting and setting of multiple roles in a single
 *      transaction, to maintain lists/counts of users and roles, and to store an optional display name
 *      for each user.
 *
+*      Originally based on an openzeppelin implementation, this implements a similar interface using bitmasks
+*      natively
+*
 *      A maximum of 256 roles are supported, and roles cannot be removed once added.
 * 
 *      Only supported roles can be queried, because this makes us "fail fast" if we make typos and it
 *      helps us to alleviate concerns around "look-a-like" strings. (See
 *      https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1090)
 */
-contract BitmaskRBAC is RBAC {
+contract BitmaskRBAC {
+  using Bitmask for uint256;
   event DisplayChanged(address indexed addr, string display);
   // TODO: event for "new supported role"
 
   struct User {
     bool exists;
     string displayName;
+    uint256 roleBitmask;
   }
 
   // This role is special. It is used to administer the RBAC itself. There is always at least one user with this role.
```

Comment:
Now that this is open source I think it's worth saying a word or two about the trust model here. Any user with RBAC_ADMIN is to some extent a trusted user of the system. Obviously they have the ability to arbitrarily change user permissions, including their own, but they also have the ability to compromise the RBAC itself in multiple and not always obvious ways:
 - they could remove the permissions of all other RBAC_ADMINs, leaving them in sole charge of the contract
 - they could add roles until there are 256 roles, at which point no more roles can ever be added (nor deleted) and any code that tries to add roles will fail
 - they could add roles that have names that look the same as existing role names, but which use different unicode characters. This approach would result in seemingly-identical roles with very different properties, and could be used to trick smart contract users or auditors into thinking the system does something it does not.

Reply:
^ yeah, we should add that comment to the README